cmd/main.go

```diff
@@ -209,17 +216,17 @@ func main() {
 		staticroutecontroller.StartStaticRouteController(mgr, commonService)
 		subnetport.StartSubnetPortController(mgr, commonService)
 
-		StartNamespaceController(mgr, commonService)
-		StartVPCController(mgr, commonService)
-		StartIPPoolController(mgr, commonService)
+		StartNamespaceController()
```

Comment:
I still suggest that we pass commonService as a argument to startcontroller instead of using one global variable.